src/app/core/common-components/entity-select/entity-select.component.ts

```diff
@@ -138,13 +138,16 @@ export class EntitySelectComponent<E extends Entity> implements OnChanges {
    * Setting the disabled state of the input element
    * @param disabled whether the input element should be disabled
    */
+  private _disabled = false;
+
   @Input() set disabled(disabled: boolean) {
-    if (disabled) {
-      this.formControl.disable();
-    } else {
-      this.formControl.enable();
-    }
+    this._disabled = disabled;
+  }
+  
+  get isDisabled(): boolean {
+    return this._disabled;
   }
```

Comment:
Did this cause any problems for you with using `formControl.disabled` in the previous implementation?

To be honest, I am not sure why we took the complex path of disabling the formControl there, but this may well have had a good reason.

Reply:
@sleidig, When `fromControl` was `disabled`, I could not make the mat-chip clickable. The `disabled` state prevented an action to be performed on the form. 

Now I am setting a boolean flag on the `Edit` button click event and checking that for enabling or disabling the input fields.

Reply:
We are including this component in complex forms, where the disabled state may indeed be set from the parent form. Therefore should not change this here to de-couple.

I remember a workaround for the clicks not firing on disabled components, that we have used for a similar problem in the EditFileComponent: setting `pointer-events: none` for the disabled input let's the clicks fall through to the parent component (see https://github.com/Aam-Digital/ndb-core/blob/2e39cec3dbc45ce8a34126080d453fe29d9a15ef/src/app/features/file/edit-file/edit-file.component.scss).
Maybe we can use this here instead.